pkg/cmd/opts/install.go

```diff
@@ -328,7 +328,7 @@ func (o *CommonOptions) InstallHelm() error {
 	binary := "helm"
 
 	if runtime.GOOS == "darwin" && !o.NoBrew {
-		err := o.RunCommand("brew", "install", "kubernetes-helm")
+		err := o.RunCommand("brew", "install", "helm@2")
```

Comment:
I am not quite sure this is going to cut it. At best this might be a solution for macOS. There is a good chance the author of #6352 is even on Linux which would mean this change is not even going to affect him (in the issue we see _/home/<user>_ opposed to _/Users/<user>_).

I think what needs to happen is:

- Remove the use of brew for installing dependencies (just jx itself is installed via Brew)
- Install dependencies in the desired version using the non-Brew approach for all OS types into ~/.jx/bin
- When calling these dependencies adjust the PATH to call our supported binary

Reply:
Will create an issue for these recommendations since it increases scope and testing required.